src/web/mjs/connectors/kuaikanmanhua.mjs

```diff
@@ -1,7 +1,7 @@
 import Connector from '../engine/Connector.mjs';
 import Manga from '../engine/Manga.mjs';
 
-export default class Kauikanmanhua extends Connector {
+export default class Kuaikanmanhua extends Connector {
```

Comment:
Rename file to reflect classname

Reply:
file is already named correctly - i just fixed the classname to reflect that!